notebooks/en/benchmarking_tgi.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<p>May be helpful to also explain some of these parameters and how they affect memory or speed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Good idea, done!</p>